src/plugins/intel_gna/src/frontend/quantization.cpp

```diff
@@ -150,7 +150,7 @@ void QuantizeWeights<int8_t>(const QuantizationData& data,
                 if (quantized_weights) {
                     value -= MAX_VAL_1B_WEIGHT;
                 } else {
-                    value = value * (data.scale_factor / channel_multiplier) + rounding_value;
+                    value = value * (data.scale_factor / ptr_int_biases[row].multiplier) + rounding_value;
```

Comment:
Wouldn't it be reasonable to add test for this change?

Reply:
It would be reasonable, but doesn't seem feasible. This should be covered by accuracy tests.